keps/prod-readiness/sig-node/1797.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 1797
+stable:
```

Comment:
Commenting here (since there are no changes to README.md)

L617: Can you please explicitly test enablement->disablement->enablement?
L633: Please answer the question (I guess it's simply "no").

Reply:
Thanks @wojtek-t . I did further testing and updated section. I also included instructions on how to check if feature is in use.